### ISSUE TITLE
problem with Microsoft SQL 

### ISSUE THREAD

**Issue by Ko-04** [controlx-io/node-red-contrib-cx-alarm-log#2]
In Node-RED, when creating a database in MSSQL, there is an issue with the reserved keyword DESC. This conflict prevents the saving of the alarm description. What are the options for using MSSQL?

**Comment by cjochum71** [comment]
Hello, I have the same problem, the problem also exists with MYSQL.
It would be great if you could help us.
Greetings, Chris

**Comment by cjochum71** [comment]
hi,
in the path:
/.node-red/node_modules/node-red-contrib-cx-alarm-log/nodes
in the file: tools.ts
search for desc and replace it with another name.
If necessary, restart node-red
Then replace desc in all nodes

**Comment by alex-controlx** [comment]
yeah, I didn't know it is a reserved word MySQL. Try this quick workaround:

![Screenshot 2024-10-27 at 11 24 20 PM](https://github.com/user-attachments/assets/33a07d80-7540-485e-8fe2-ac08a5d591ef)

the `MySQL converter` function is the following:
```javascript
renameDescToDescription(msg.payload.toAdd);
renameDescToDescription(msg.payload.toUpdate);

return msg;

function renameDescToDescription(arr) {
    for (const event of arr) {
        if (event.desc != null) {
            event.description = event.desc;
            delete event.desc;
        }
    }
}
```
